docs-chef-io/content/inspec/resources/aws_api_gateway_v2_integration.md

```diff
@@ -0,0 +1,171 @@
+AWS_PROXY: for integrating the route or method request with a Lambda function or other AWS service action. This integration is also referred to as a Lambda proxy integration.
+HTTP: for integrating the route or method request with an HTTP endpoint. This integration is also referred to as the HTTP custom integration. Supported only for WebSocket APIs.
+HTTP_PROXY: for integrating the route or method request with an HTTP endpoint, with the client request passed through as-is. This is also referred to as HTTP proxy integration.
+MOCK: for integrating the route or method request with API Gateway as a "loopback" endpoint without invoking any backend. Supported only for WebSocket APIs.
+
+`integration_uri`
+: For a Lambda integration, specify the URI of a Lambda function.
+For an HTTP integration, specify a fully-qualified URL.
+For an HTTP API private integration, specify the ARN of an Application Load Balancer listener, Network Load Balancer listener, or AWS Cloud Map service. If you specify the ARN of an AWS Cloud Map service, API Gateway uses DiscoverInstances to identify resources. You can use query parameters to target specific resources. To learn more, see DiscoverInstances . For private integrations, all resources must be owned by the same AWS account.
+
+`passthrough_behavior`
+: Specifies the pass-through behavior for incoming requests based on the Content-Type header in the request, and the available mapping templates specified as the requestTemplates property on the Integration resource. There are three valid values: WHEN_NO_MATCH, WHEN_NO_TEMPLATES, and NEVER. Supported only for WebSocket APIs.
+WHEN_NO_MATCH passes the request body for unmapped content types through to the integration backend without transformation.
+NEVER rejects unmapped content types with an HTTP 415 Unsupported Media Type response.
+WHEN_NO_TEMPLATES allows pass-through when the integration has no content types mapped to templates. However, if there is at least one content type defined, unmapped content types will be rejected with the same HTTP 415 Unsupported Media Type response.
+
+`payload_format_version`
+: Specifies the format of the payload sent to an integration. Required for HTTP APIs.
+
+`request_parameters`
+: For WebSocket APIs, a key-value map specifying request parameters that are passed from the method request to the backend. The key is an integration request parameter name and the associated value is a method request parameter value or static value that must be enclosed within single quotes and pre-encoded as required by the backend. The method request parameter value must match the pattern of method.request.{location}.{name}, where {location}is querystring, path, or header; and {name}must be a valid and unique method request parameter name.
+
```

Comment:
A doubt - dont we have to mention abour response parameter?

Reply:
Yeah... I have added it. Please check.

Reply:
ok